EnsureGoPath fails when GOPATH has multiple directories listed

So, if your GOPATH has multiple directories listed, the new `EnsureGoPath` function fails when calling a stat on `$GOPATH/src`.  e.g.

```
> export GOPATH=/tmp/foo/.go:/tmp/foo
> GO15VENDOREXPERIMENT=1 glide install
[ERROR] Could not find /tmp/foo/.go:/tmp/foo/src.
[INFO] As of Glide 0.5/Go 1.5, this is required.
[ERROR] Wihtout src, cannot continue. stat /tmp/foo/.go:/tmp/foo/src: no such file or directory
```

This was introduced with [the latest commit](https://github.com/Masterminds/glide/commit/87489e82b1ac46345089552b7a6736c10f582ece#diff-fbd61ea1f28a9198f93880a8e505bc37R90).

--- a/action/ensure.go
+++ b/action/ensure.go
@@ -5,6 +5,7 @@ import (
 	"os"
 	"os/exec"
 	"path"
+	"strings"
 
 	"github.com/Masterminds/glide/cfg"
 	"github.com/Masterminds/glide/msg"
@@ -83,15 +84,22 @@ func EnsureVendorDir() {
 //
 // Otherwise it returns the value of GOPATH.
 func EnsureGopath() string {
-	gp := os.Getenv("GOPATH")
-	if gp == "" {
+	gps := gpath.Gopaths()
+	if len(gps) == 0 {
 		msg.Die("$GOPATH is not set.")
 	}
-	_, err := os.Stat(path.Join(gp, "src"))
-	if err != nil {
-		msg.Error("Could not find %s/src.\n", gp)
-		msg.Info("As of Glide 0.5/Go 1.5, this is required.\n")
-		msg.Die("Wihtout src, cannot continue. %s", err)
+
+	for _, gp := range gps {
+		_, err := os.Stat(path.Join(gp, "src"))
+		if err != nil {
+			msg.Warn("%s", err)
+			continue
+		}
+		return gp
 	}
-	return gp
+
+	msg.Error("Could not find any of %s/src.\n", strings.Join(gps, "/src, "))
+	msg.Info("As of Glide 0.5/Go 1.5, this is required.\n")
+	msg.Die("Wihtout src, cannot continue.")
+	return ""
 }

